v2.1/create-security-certificates-custom-ca.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Create Security Certificates
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
+toc: true
+---
+
+A secure CockroachDB cluster uses TLS 1.2 for encrypted inter-node and client-node communication and requires a Certificate Authority (CA) certificate as well as keys and certificates for nodes, clients, and (optionally) the Admin UI. To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, use [`openssl` commands](https://wiki.openssl.org/index.php/), or use a custom CA (for example, a public CA or your organizational CA).
```

Comment:
It looks like this is a slight rewrite of the original intro, so I think it's worth improving it a bit (feel free to disregard). I think that the doc might be better served by talking about the user's experience/needs, rather than what the product does/needs. As a straw man:

> To secure your CockroachDB cluster's inter-node and client-node communication, you need to provide a Certificate Authority (CA) certificate that has been used to sign keys and certificates (SSLs) for:
> - Nodes
> - Clients
> - Admin UI _(optional)_

This is a subtle change, but I think it helps users more clearly understand who needs to do what.